### PR TITLE
View bug fix + refactoring

### PR DESCRIPTION
**Pull request by ljelinkova**
- fixed issue when it was not possible to open view + close view + open view again (widget is disposed exception)
- changed view class name
- changed view id in plugin.xml

### CODE REVIEW

**Comment by ljelinkova** [comment]
This should fix #27 and #24

File: plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/src/com/redhat/fabric8analytics/lsp/eclipse/ui/StackAnalysesView.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	@Override
+	public void setFocus() {
+		browser.setFocus();
+	}
+
+	public void updatebrowserUrl(String browseUrl) {
+		Display.getDefault().asyncExec(new Runnable(){
+			public void run(){
+				if (!browser.isDisposed()) {
+					browser.setUrl(browseUrl);
```
Comment (GeetikaBatra) [inline plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/src/com/redhat/fabric8analytics/lsp/eclipse/ui/StackAnalysesView.java]: @ljelinkova if the browser gets disposed off, would this still work or is that case always covered?

Comment (ljelinkova) [reply plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/src/com/redhat/fabric8analytics/lsp/eclipse/ui/StackAnalysesView.java]: If the browser gets disposed the previous code would have thrown an exception, this check should prevent it.

**Comment by GeetikaBatra** [comment]
@ljelinkova, Can you please rebase this branch. Thanks in advance.

**Comment by ljelinkova** [comment]
I've created a new PR for this ( #46).

**Pull request closed without merging**